bin/utils_test.py

```diff
@@ -21,6 +21,42 @@
 [connectors]
 sync_exec=/usr/libexec/argo-egi-connectors
 sync_path=/var/lib/argo-connectors
+
+[sampling]
+
+s_period=1440
+s_interval=5
+
+
+[datastore_mapping]
+
+e_map=dt,ap,p,s,n,hsp,a,r,up,u,d,m,pr,ss,cs,i,sc
+s_map=dt,ap,p,s,n,sf,a,r,up,u,d,m,pr,ss,cs,i,sc
+sd_map=ts,s,sum,msg,ps,pts,di,ti
+n_alt=vo_f
+"""
+
+TEST_TENANT_DB_CONTENTS = r"""
+{
+  "db_conf": [
+    {
+      "store": "ar",
+      "server": "192.168.0.99",
+      "port": 27017,
+      "database": "argo-egi",
```

Comment:
Better change `argo-egi` to `argo_EGI` (the same name we have it on test data on devel API node).